src/third_party/starboard/wpe/rpi/configuration_public.h

```diff
@@ -361,7 +361,7 @@
 // video.
 #define SB_MEDIA_MAX_VIDEO_BITRATE_IN_BITS_PER_SECOND (200 * 1024 * 1024)
 
-#define SB_MEDIA_MAX_VIDEO_FRAMERATE_IN_FRAMES_PER_SECOND (24)
+#define SB_MEDIA_MAX_VIDEO_FRAMERATE_IN_FRAMES_PER_SECOND (30)
```

Comment:
Ok but keep in mind this was also done with vp9 in mind.

Reply:
OK. Thanks @pawel-stanek-ml